README.md

```diff
@@ -0,0 +1,56 @@
+# BBSTrader
+
+### QUICKSTART
+1. `git clone https://github.com/BensGitHub2022/MT5-Expert-Advisor.git`
+2. `cd MT5-Expert-Advisor`
+3. `python -m venv .venv`
+4. `source .venv/Scripts/activate`
+5. `pip install -r requirements.txt`
+6. Setup pkg/credentials.json. More on this [here](#set-your-mt5-account-credentials-in-credentialsjson).
+7. `python main.py`
+ 
+## Dependencies
+
+### Install Python
+BBSTrader requires [Python](https://www.python.org/downloads/) version 3.6-3.10. Note: Python version 3.11 & 3.12 are not supported by MT5!
```

Comment:
We now need to be at 3.10 in order to use the match-case.

Reply:
Fixed